cloud_testing/kotlin_poc/src/main/kotlin/ftl/android/AndroidCatalog.kt

```diff
@@ -0,0 +1,36 @@
+package ftl.android
+
+import ftl.gc.GcTesting
+
+
+/**
+ * Contains lists of possible Android device and version ids, as well as checks
+ * to validate device configuration
+ */
+object AndroidCatalog {
+    private val androidDeviceCatalog by lazy {
+        GcTesting.get.testEnvironmentCatalog().get("android").execute().androidDeviceCatalog
+    }
+
+    val androidModelIds by lazy { androidDeviceCatalog.models.map { it.id } }
+    val androidVersionIds by lazy { androidDeviceCatalog.versions.map { it.id } }
```

Comment:
do we need?

- androidModelIds
- androidVersionIds

it seems like ` androidDeviceCatalog.models.find ` covers the validation?

Reply:
nevermind, it looks like the error messages are a bit cleaner with this.